app/docs/0.13.x/plugin-development/plugin-configuration.md

```diff
@@ -23,9 +23,8 @@ Your plugin's configuration is being verified against your schema when a user is
 For example, a user performs the following request:
 
 ```bash
-$ curl -X POST http://kong:8001/plugins/ \
+$ curl -X POST http://kong:8001/services/my-service/plugins/ \
```

Comment:
Seems like we should use `/services/<service-name-or-id>/plugins` here to stay consistent (and also self-document the "by-id" endpoint

Reply:
I tried to be consistent with `my-custom-plugin` in the line below it. Your suggestion works too